content/docs/setup/kubernetes/additional-setup/config-profiles/index.md

```diff
@@ -53,12 +53,17 @@ The components marked as **X** are installed within each profile:
 Some profiles have an authentication variant, with `-auth` appended to the name, which adds the following
 security features to the profile:
 
-| | default | demo | minimal | sds |
+| | default | demo | minimal | sds-auth |
```

Comment:
for the first field, you might indicate `Security Model` or something equivalent.